mmv1/third_party/terraform/tests/resource_bigtable_gc_policy_test.go

```diff
@@ -165,6 +165,35 @@ func TestAccBigtableGCPolicy_gcRulesPolicy(t *testing.T) {
 	})
 }
 
+func TestAccBigtableGCPolicy_updatePolicyGCRulesUpdated(t *testing.T) {
+	// bigtable instance does not use the shared HTTP client, this test creates an instance
+	skipIfVcr(t)
+	t.Parallel()
+
+	instanceName := fmt.Sprintf("tf-test-%s", randString(t, 10))
+	tableName := fmt.Sprintf("tf-test-%s", randString(t, 10))
+	familyName := fmt.Sprintf("tf-test-%s", randString(t, 10))
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckBigtableGCPolicyDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccBigtableGCPolicy_gcRulesCreate(instanceName, tableName, familyName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccBigtableGCPolicyExists(
+						t, "google_bigtable_gc_policy.policy"),
+					testAccBigtableCanUpdatePolicy(
+						t, "google_bigtable_gc_policy.policy", bigtable.MaxVersionsPolicy(2)),
+					resource.TestCheckResourceAttr(
```

Comment:
@kevinsi4508 I haven't found a way to force `Read` before checking this attribute. Does `resource` trigger Read automatically ?